### PR TITLE
update conditions.md - every func

### PR DESCRIPTION
**Pull request by HighHarmonics2**
Added note on how to silence a full cycle with the every function.

### CODE REVIEW

File: docs/reference/conditions.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ For example, `every' 3 0 (fast 2)` will speed up the cycle on cycles 0,3,6,… w
 
 With this in mind, setting the second argument of `every'` to `0` gives the equivalent `every` function. For example, `every 3` is equivalent to `every' 3 0`.
 
+**Note**: The `every` functions applys to other functions but not directly on the cycle. You can use `every silent` or `every mask ("~")` to silence a full cycle. Mask gives additional flexibility to turn on/off individual steps.
```
Comment (Zalastax) [inline docs/reference/conditions.md:45]: Typo: applys -> apply (or applies if it had been singular)

Unfortunately I don't understand "The `every` functions applys to other functions but not directly on the cycle."

**Review by Zalastax — COMMENTED** [review]
Thanks a lot for the contribution!

I think this is a neat concept which would be nice to have in the documentation, but I am not sure this would be the best place in the long run because the example doesn't bring more clarity to the `every` functions per se. I wish we had a place for various cool examples. For now, I think it's okay and I think we should merge after my other commitments are addressed.

File: docs/reference/conditions.md (hunk 1 above)
Comment (HighHarmonics2) [reply docs/reference/conditions.md:45]: Thanks for the feedback and for checking my spelling (oops) and wording. I'm trying to say that you can't use "every x" to control the cycle directly. But I agree that my sentence isn't clear. 

Here is an alternative - make this a "tip" - which makes it stand out from the "every" functions. Then delete the unclear sentence and have this:

:::tip: The `every` functions can be used to silence a full cycle or part of a cycle by using `silent` or `mask ("~")`. Mask gives additional flexibility to turn on/off individual steps.

d1 $ every 3 silent $ n "2 9 11 2" # s "hh27"
d1 $ every 3 (mask "~") $ n "2 9 10 2" # s "hh27"
d1 $ every 3 (mask "0 0 0 0") $ n "2 9 11 2" # s "hh27" :::

If this is OK, how should I proceed to make this change?

Comment (jqueraltmo) [reply docs/reference/conditions.md:45]: Just add a new commit to your branch, it should appear here automatically as long as the merge request is open.

"I wish we had a place for various cool examples." Well, we should create a new "Recipes" or "Tips and tricks" file inside the Tutorials section, and compile these ideas.

Comment (HighHarmonics2) [reply docs/reference/conditions.md]: Thanks @jqueraltmo. I added a new commit with the change. I like your idea of a Tips section. But I also think there is still a place for some tips embedded with the Reference topic, particularly if it is very specific - a tip on how to use a function. 

I will resolve this conversation.

**Comment by Zalastax** [comment]
Lovely! Really good job on this 💯

**Pull request merged**